### DISCUSSION TITLE
关闭 [BUG] macos 每次重启后会打开都会提示“'osascript'想要进行更改”

### DISCUSSION THREAD

**Discussion by Drom1145** [clash-verge-rev/clash-verge-rev#2216] [Q&A] [answered]
问题：mac重启，重新登陆并打开（或者开启自启），会发现提示“'osascript'想要进行更改”，每次重启都会提示且输入密码，感到繁琐。
下面是我发生问题的截屏画面
<img width="345" alt="截屏2024-11-28 18 03 41" src="https://github.com/user-attachments/assets/877af5e2-4d8d-416a-86da-a16c61add91d">

在此之前，我把这个问题当作bug处理并进行上诉，结果开发人员认为我电脑安装了安全软件。
<img width="1094" alt="截屏2024-11-30 12 43 23" src="https://github.com/user-attachments/assets/58b9dfce-a1e4-4036-8ed6-89e0094c8b97">
如图所示的结果，不过我并没有执行远程排查，所以我对这个检测结果是相当怀疑的。
所以得到消息后我排查电脑中的程序，还开了新的电脑用户重新安装，发现还是出现此问题，说明很有可能是软件问题。

想要得到一个解决方案，感谢各位大佬了

**Comment by baigongli** [comment]
这个不应该是 app 执行什么指令，然后要提权，mac 弹窗出来？

**Reply by Drom1145** [reply]
> 这不应该是app执行什么指令，然后要提权，mac弹窗出来？

是这款软件导致的，因为不授权就打不开

**Comment by Drom1145** [comment, marked as answer]
开发者已在最新版v2.0.2中修复了此bug，若还有问题可以联系开发者

**Reply by YanYiGe2023** [reply]
> 开发者已在最新版v2.0.2中修复了此bug，若还有问题可以联系开发者

根本没有修复，我也是macOS系统，v2.0.2版本，一样开机提示

**Reply by Drom1145** [reply]
> > 开发者已在最新版v2.0.2中修复了此bug，若还有问题可以联系开发者
> 
> 根本没有修复，我也是macOS系统，v2.0.2版本，一样开机提示

有人指出”允许在后台中“wo fen打开就没事了，如果不行那你自己开个楼吧毕竟我已经解决了
